### ISSUE TITLE
Types not found when importing from 'jimp/es'

### ISSUE THREAD

**Issue by rcoundon** [jimp-dev/jimp#1206]
# Expected Behavior
Types to resolve when using the ES build

# Current Behavior
The TypeScript compiler complains that it cannot find the declaration file when importing from jimp/es

# Failure Information (for bugs)

```
 Could not find a declaration file for module 'jimp/es'.
 Try `npm i --save-dev @types/jimp` if it exists or add a new declaration (.d.ts) file containing `declare module 'jimp/es';`

import Jimp from 'jimp/es';
```

## Steps to Reproduce
Simply add
```
import Jimp from 'jimp/es';
```

- Jimp Version: 0.22.5
- Operating System: MacOS 13.2.1 (22D68)
- Node version: v16.19.0

**Comment by hipstersmoothie** [comment]
@crutchcorn we could probably fix this before the TS rewrite

**Comment by KodeMunkie** [comment]
Is there a workaround for this?

**Comment by hipstersmoothie** [comment]
:rocket: Issue was released in [`v1.0.2`](https://github.com/jimp-dev/jimp/releases/tag/v1.0.2) :rocket:
